pkg/restapi/operation/common/operation.go

```diff
@@ -37,10 +41,21 @@ const (
 	registerMsgService    = msgServiceOperationID + "/register-service"
 	unregisterMsgService  = msgServiceOperationID + "/unregister-service"
 	msgServiceList        = msgServiceOperationID + "/services"
+	sendNewMsg            = msgServiceOperationID + "/send-new-message"
```

Comment:
should we be more succinct? e.g., send and reply?

Reply:
send-message & reply-message?

Reply:
done